internal/pkg/manifest/validate_env.go

```diff
@@ -198,3 +207,21 @@ func (o privateHTTPConfig) Validate() error {
 	}
 	return nil
 }
+
+func (c EnvironmentConfig) validateSubnets() error {
```

Comment:
nit:
```suggestion
func (c EnvironmentConfig) validateInternalALBSubnets() error {
```

Reply:
I did this bc in the manifest the field will just be `subnets` but you're right, this method should be more clearly named.